libvips/foreign/foreign.c

```diff
@@ -2271,6 +2272,8 @@ vips_heifload_source( VipsSource *source, VipsImage **out, ... )
  * Use @bitdepth to set the bitdepth of the output file. HEIC supports at
  * least 8, 10 and 12 bits; other codecs may support more or fewer options.
  *
+ * Use @encoder if you want to use a specific encoder (e.g. aom, svt etc).
```

Comment:
I think this is a little unclear. Suggestion:

- In the "compression" paragraph above, change to "Use @compression to set the compression format, e g. ..."
- Change this text to be "Use @encoder to set the encode library, eg. aom, rav1e, etc."